addons/binding/org.openhab.binding.co2signal/README.md

```diff
@@ -0,0 +1,110 @@
+# CO2 Signal Binding
+
+Inspired by Henrik Knibergs attempt to save the climate http://climate.crisp.se/ here is a a binding that tries to help reduce the CO2 footprint via OpenHab
+
+This binding uses the [co2signal.com service](https://www.co2signal.com) for providing CO2 Signal information for any location worldwide.
+
+By using electricity at the right time, your device favors low-carbon sources of electricity
+
+The project is providing informantion on how clean the current electricity is produced, via those two websites: [co2signal.com](http://www.co2signal.com) and [www.electricitymap.org](https://www.electricitymap.org).
+
+To use this binding, you first need to [register and get your API token](https://www.co2signal.com/).
+
+Credits go to the authors of the airquality binding, most code was copied from there.
+
+## Supported Things
+
+There is exactly one supported thing type, which represents the CO2 Signal information for an observation location. It has the `co2signal` id. Of course, you can add multiple Things, e.g. for measuring co2 for different locations.
+
+## Discovery
+
+There is no discovery implemented. You have to create your things manually.
+
+## Binding Configuration
+ 
+The binding has no configuration options, all configuration is done at Thing level.
+ 
+## Thing Configuration
+
+The thing has a few configuration parameters:
+
+| Parameter | Description                                                              |
+|-----------|------------------------------------------------------------------------- |
```

Comment:
You would make me happy by formatting the tables in the MD file as well :-D

http://www.tablesgenerator.com/markdown_tables

Select 'File' > 'Past table data ...' and past the unformatted mark down table, you can then copy paste the formatted table 👍

Reply:
Fixed